collector/Makefile

```diff
@@ -48,12 +48,12 @@ package: build
 
 .PHONY: publish
 publish:
-	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/collector-extension-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
+	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/opentelemetry-collector-layer-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
```

Comment:
I see that the name for the file produced in the [.PHONY: package](https://github.com/open-telemetry/opentelemetry-lambda/pull/701/files#diff-3a4b47f3a82c1fff6fe963bf57a89a541a38197bb644cdd8616b576c513b7d40R47) is `opentelemetry-collector-layer-$(GOARCH).zip`. Is this fixing an issue where the `.PHONY: publish` action was failing because it couldn't find the file to publish? :thinking:

Reply:
Yes That is correct and it will publish now with this change, it wasn't able to find the file since it was renamed.